.github/workflows/build.yml

```diff
@@ -26,10 +26,29 @@ jobs:
       run: |
         sudo apt-get -y install pandoc
     - uses: actions/checkout@v2
+      with:
+        fetch-depth: 10
```

Comment:
what is this? can you add a comment explaining it

Reply:
and why is 10 sufficient?

Reply:
Added a comment.